server/routes/projects.ts

```diff
@@ -0,0 +1,154 @@
+import * as express from 'express';
+import { UserRequest } from '../types';
+
+import { PoolClient } from 'pg';
+import { pgPool } from 'common/pool';
+import { NotFoundError, UnauthorizedError, ForbiddenError } from '../errors';
+import { getIsProjectAdmin } from './posts';
+import { bucketName, deleteFile } from './files';
+
+const router = express.Router();
+
+// DELETE project and dependent resources by id
+router.delete('/', async (req: UserRequest, res, next) => {
+  let client: PoolClient | null;
+  try {
+    client = await pgPool.connect();
+    const { id: projectId } = req.body;
+    const { accountId } = req.user;
```

Comment:
this causes a TS error because `req.user` can be undefined
```suggestion
    const accountId = req.user?.accountId;
```